browser/ui/content_settings/brave_autoplay_content_setting_bubble_model.cc

```diff
@@ -33,14 +37,16 @@ BraveAutoplayContentSettingBubbleModel::BraveAutoplayContentSettingBubbleModel(
   SetRadioGroup();
 }
 
-BraveAutoplayContentSettingBubbleModel::~BraveAutoplayContentSettingBubbleModel() {}
+BraveAutoplayContentSettingBubbleModel::
+~BraveAutoplayContentSettingBubbleModel() {}
 
 void BraveAutoplayContentSettingBubbleModel::CommitChanges() {
   if (settings_changed()) {
     ContentSetting setting = selected_item() == kAllowButtonIndex
                                  ? CONTENT_SETTING_ALLOW
                                  : block_setting_;
     SetNarrowestContentSetting(setting);
+    web_contents()->GetController().Reload(content::ReloadType::NORMAL, true);
```

Comment:
do we need reload when changing from allow to block?

Reply:
I think that case could not be happened via contents settings bubble dialog because autoplay content settings icon is not visible when user enable it.
Instead, user can change it by other route.(ex, site info dialog)